docs/html/development/getting-started.rst

```diff
@@ -123,8 +123,22 @@ To build it locally, run:
 
 The built documentation can be found in the ``docs/build`` folder.
 
+What Next?
+==========
+
+The following pages may be helpful for new contributors on where to look next
+in order to start contributing.
+
+* Some `good first issues`_ on Github for new contributors
```

Comment:
```suggestion
* Some `good first issues`_ on GitHub for new contributors
```

Reply:
Thanks for the comment @uranusjr. I fixed the casing and squashed all commits into one. Please take a look, and if it looks good, feel free to approve and merge the PR